src/plugins/bookPlayer/plugin.js

```diff
@@ -1,14 +1,14 @@
-import browser from 'browser';
-import loading from 'loading';
-import keyboardnavigation from 'keyboardnavigation';
-import dialogHelper from 'dialogHelper';
-import dom from 'dom';
-import events from 'events';
-import 'css!./style';
-import 'material-icons';
-import 'paper-icon-button-light';
-
+import loading from '../../components/loading/loading';
+import keyboardnavigation from '../../scripts/keyboardNavigation';
+import dialogHelper from '../../components/dialogHelper/dialogHelper';
+import '../../scripts/dom';
+import { Events } from 'jellyfin-apiclient';
+import './style.css';
+import 'material-design-icons-iconfont';
+import '../../elements/emby-button/paper-icon-button-light';
+import ServerConnections from '../../components/ServerConnections';
 import TableOfContents from './tableOfContents';
+import browser from '../../scripts/browser';
```

Comment:
If you move these two around the diff would look much better but it's not really important.

Reply:
if it's not important, i will leave it as is